### PR TITLE
README.md: Setup details, reflow.

### PR DESCRIPTION
**Pull request by ilya-bobyr**
As we expect more people to use our CLI, it is good to provide more
details as to how to set it up and run the commends.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Prettier will format everything, except what is explicitly specified here.
+
+# npm
+node_modules/
```
Comment (Alexsey) [inline .prettierignore:4]: prettier would ignore `node_modules` folder by default, so you don't need to list it in `.prettierignore`. See [File patterns](https://prettier.io/docs/en/cli.html#--ignore-path) section

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+# Generated files
+dist/
+
+# Configuration
+.env
```
Comment (Alexsey) [inline .prettierignore:10]: `.env` wouldn't be formatted by prettier - they have explicitly [decided not to do it](https://github.com/prettier/prettier/issues/6103. So there is no need to list it in `.prettierignore` as well

**Review by gerben-stavenga — APPROVED** [review]

File: .prettierignore (hunk 1 above)
Comment (ilya-bobyr) [reply .prettierignore:4]: Ok, thank you.

File: .prettierignore (hunk 2 above)
Comment (ilya-bobyr) [reply .prettierignore:10]: Not sure that relying on an explicit rule for this one is as obvious as it is for `node_modules`.
We have it in our `.gitignore`, so it seems easier if `.gitingore` and `.prettierignore` are identical.

Comment (ilya-bobyr) [reply .prettierignore]: Replaced with a comment saying that `.env` is excluded by an implicit rule.

**Pull request merged**